goodtables/cells.py

```diff
@@ -1,13 +1,17 @@
 from six.moves import zip_longest
 
 
-def create_cells(headers, schema_fields, values=None, row_number=None):
+def create_cells(headers, schema_fields, optional_fields, values=None, row_number=None):
     """Create list of cells from headers, fields and values.
 
     Args:
         headers (List[str]): The headers values.
         schema_fields (List[tableschema.field.Field]): The tableschema
             fields.
+        optional_fields (List[str]): List of schema fields which are defined as
```

Comment:
If we look for a list of strings than we need to use something like `optional_field_names` to prevent confusion with the `field` concept (an object/dict)